### PR TITLE
Use kubevirt ansible roles

### PR DESCRIPTION
**Pull request by rthallisey**
We'll use ansible-galaxy to pull in kubevirt-ansible and run the kubevirt playbook.

### CODE REVIEW

**Comment by rthallisey** [comment]
Changing this to build time

**Comment by karmab** [comment]
maybe it can just be pushed already ?

File: README.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ As the resources deployed need cluster-admin privileges, credentials of a super
 #### Deploy from the UI
```
Comment (lukas-bednar) [inline README.md]: This typo `s/UI/CLI/` is like disease, spreading across projects  :-D ...

Comment (karmab) [reply README.md]: you made me laugh

**Comment by rthallisey** [comment]
@karmab I haven't tested in the UI, merging

**Pull request merged**